tools/depends/configure.ac

```diff
@@ -97,6 +97,10 @@ AC_ARG_ENABLE([wayland],
   [enable_wayland=$enableval],
   [enable_wayland=no])
 
+AC_ARG_WITH([rendersystem],
+  [AS_HELP_STRING([--with-rendersystem],
+  [render system to use])],
+  [app_rendersystem=$withval])
```

Comment:
we should probably add a content sanity check for this option

Reply:
just sanity for gl vs gles or actually checking if value is valid for platform in question ?

Reply:
gl/gles check should be enough imho

Reply:
added.